test/vorpal.js

```diff
@@ -108,6 +108,16 @@ describe('argument parsing', function () {
     obj(vorpal.execSync('required bar')).should.equal(fixture);
   });
 
+  it('should execute a command with a required arg which has double quote', function () {
+    var fixture = obj({ options: {}, str: '{"test": 1}' });
+    obj(vorpal.execSync('required \'{"test": 1}\'')).should.equal(fixture);
+  });
+
+  it('should execute a command with a required arg which has single quote', function () {
+    var fixture = obj({ options: {}, str: "{'test': 1}" });
+    obj(vorpal.execSync('required "{\'test\': 1}"')).should.equal(fixture);
+  });
```

Comment:
Tests are passing somehow?

Reply:
it seems like these tests are not very useful =(